pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -223,6 +224,17 @@ type ACMEIssuerDNS01ProviderAcmeDNS struct {
 	AccountSecret SecretKeySelector `json:"accountSecretRef"`
 }
 
+// ACMEIssuerDNS01ProviderInfoblox is a structure containing the
+// configuration for Infoblox DNS servers
+type ACMEIssuerDNS01ProviderInfoblox struct {
+	GridHost           string            `json:"gridHost"`
+	WapiUsername       string            `json:"wapiUsername"`
+	WapiPasswordSecret SecretKeySelector `json:"wapiPasswordSecret"`
+	WapiPort           int               `json:"wapiPort"`
+	WapiVersion        string            `json:"wapiVersion"`
+	SslVerify          bool              `json:"sslVerify"`
```

Comment:
I'll change. I think we need that because there may be guys just testing this combo(cert-manager with Infoblox) without having a correct certificate on their Infoblox instance. What do you think?